client-dotnet/src/EvoMaster.Instrumentation/Coverage/MethodReplacement/DistanceHelper.cs

```diff
@@ -0,0 +1,219 @@
+using System;
+
+public class DistanceHelper
+{
+    public static readonly double H_REACHED_BUT_NULL = 0.05d;
+
+    public static readonly double H_NOT_NULL = 0.1d;
+
+    public static readonly double H_REACHED_BUT_EMPTY = H_REACHED_BUT_NULL;
+
+    public static readonly double H_NOT_EMPTY = H_NOT_NULL;
+
+
+    //2^16=65536, max distance for a char
+    public static readonly int MAX_CHAR_DISTANCE = 65_536;
```

Comment:
does this apply to .Net as well? if so, add a comment, eg "Like on JVM, also in .Net a char..."